docs/src/model_creation/network_analysis.md

```diff
@@ -187,22 +187,18 @@ N == Z*B
 ```
 
 Reaction complexes give an alternative way to visualize a reaction network
-graph. Catalyst's [`complexgraph`](@ref) command will calculate the complexes of
+graph. Catalyst's [`plot_complexes`](@ref) command will calculate the complexes of
 a network and then show how they are related. For example,
-```julia
-complexgraph(rn)
+```@example s1
+using Catalyst, GraphMakie, GLMakie
+plot_complexes(rn)
 ```
-gives
-
-![Simple example complex graph](../assets/simple_complexgraph.svg)
 
 while for the repressilator we find
-```julia
-complexgraph(repressilator)
+```@example s1
+plot_complexes(repressilator)
```

Comment:
This graph looks quite mess due to the Hill functions. Can you disable the labels on the lines for it?